gradle.properties

```diff
@@ -52,7 +52,7 @@ publicNexusLocation="http://nexus.h2o.ai:8081/repository"
 httpClientVersion=4.5.2
 
 # Version of Apache Parquet dependency (should be kept in sync with the version used in current Spark releases)
-defaultParquetVersion=1.12.3
+defaultParquetVersion=1.8.1
```

Comment:
Will have to revert upgrade of parquet library. Some tests on older hadoop tests fail. I thought the upgrade is releated only to standalone and apparently not.